### PR TITLE
notifications-group-select-list-item-cy-219

### PR DESCRIPTION
**Pull request by steiningermona**

### CODE REVIEW

**Comment by steiningermona** [comment]
**from figma**
image
**from storybook**
image
hover:
image
active:
image
activated:
image

File: src/components/notifications-group-select-list-item/NotificationsGroupSelectListItem.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+export const NotificationsGroupSelectListItem: React.FC<INotificationsGroupSelectListItemProps> = ({ title, activated, previewProps }) => {
+  return (
+    <SNotificationsGroupSelectListItem activated={activated}>
+      <SFlexListItem>
+        <Preview size={28} {...previewProps}
+        ></Preview>
+        <SNotificationsGroupSelectListItemTitle>{title}</SNotificationsGroupSelectListItemTitle>
+      </SFlexListItem>
+      <IconChevronDown></IconChevronDown>
```
Comment (diegomart2000) [inline src/components/notifications-group-select-list-item/NotificationsGroupSelectListItem.tsx]: @steiningermona 

[] If your component has a value it also needs a way to set that value, pass an onChange event.

--- In this case, the ChevronDown needs a onClick handler to be passed to notify parent to activate this item.

File: src/components/notifications-group-select-list-item/NotificationsGroupSelectListItem.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  height: 100%;
+`;
+
+export const NotificationsGroupSelectListItem: React.FC<INotificationsGroupSelectListItemProps> = ({ title, activated, previewProps }) => {
+  return (
+    <SNotificationsGroupSelectListItem activated={activated}>
+      <SFlexListItem>
+        <Preview size={28} {...previewProps}
+        ></Preview>
+        <SNotificationsGroupSelectListItemTitle>{title}</SNotificationsGroupSelectListItemTitle>
```
Comment (diegomart2000) [inline src/components/notifications-group-select-list-item/NotificationsGroupSelectListItem.tsx:32]: @steiningermona Multiple words with white spaces break the component.

**Review by diegomart2000 — CHANGES_REQUESTED** [review]

**Comment by steiningermona** [comment]
**from figma**
image
**from storybook**
image
activated:
image

**Review by diegomart2000 — APPROVED** [review]

**Pull request merged**